jenkins/debian/debos/scripts/buster-xfstests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+#####################################################
+# This script create buster image for xfstests
+#####################################################
+
+set -e
+
+# Install required xfstests packages
+apt-get install --no-install-recommends xfslibs-dev uuid-dev libtool-bin \
+	        e2fsprogs automake gcc libuuid1 quota attr libattr1-dev make \
+	        libacl1-dev libaio-dev xfsprogs libgdbm-dev gawk fio dbench \
+	        uuid-runtime python sqlite3 git bc bsdmainutils acl -y
+
+apt-get install perl hostname net-tools -y
+
+# Clone repo
+git clone git://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git /xfstests
```

Comment:
Please store the URL and revision in a local manifest file so it can be added to the rootfs build-info.json.

Reply:
made this change and tested.